### ISSUE TITLE
Give an option to disable Ltex on specific file in workspace

### ISSUE THREAD

**Issue by tonydavidx** [valentjn/vscode-ltex#576]
It would be nice if there is an option to disable Ltex automatic diagnose on certain files

**Comment by pixelpusher** [comment]
Yes! This is a huge problem when you try and open a bibliography. Would be great to ignore file types in a workspace

**Comment by SethFalco** [comment]
In general, I would like to see it work with wildcards, so one can specify files, file types, directories, etc.

It should support a similar syntax as the search inclusions/exclusions in the VSC Find and Replace sidebar.

i.e.
* `file.txt`
* `*.txt`
* `directory/`
* `directory/**/*.txt`

**Comment by chrisgrieser** [comment]
agree. As with many linters, this could be a `.ltexignore` file which also uses `.gitignore` syntax
